get_started_with_gx_cloud_and_postgres/workshop.md

```diff
@@ -0,0 +1,204 @@
+5. [Validate Expectations](#validate-a-data-asset)
+6. [Update the failing Expectation and run the Validation again](#update-the-failing-expectation-and-run-the-validation-again)
+7. [Fetch Metrics](#fetch-metrics)
+
+
+## GX terminology
+If you're new to GX, an understanding of the following GX terminology will be helpful as you complete this workshop.
+
+<img src="../common/img/gx_terminology.png" alt="Introductory GX terminology" style="width:800px;"/><br>
+
+## Sign in to GX Cloud
+Sign in to [GX Cloud](https://hubl.li/Q02ng2Jx0).
+
+## Run the GX Agent and Postgres database
+The GX Agent is an intermediary between GX Cloud and your data stores. GX Cloud does not connect directly to your data; all data access occurs within the GX Agent. The GX Agent receives jobs from GX Cloud, executes these jobs against your data, and then sends the job results back to GX Cloud. The GX Agent runs in an environment where it has access to your data. Today, you'll run it on your local machine using Docker.
+
+To learn more about the GX Agent and how it works with GX Cloud, [see our GX Cloud architecture documentation](https://docs.greatexpectations.io/docs/cloud/about_gx#gx-cloud-architecture).
+
+### Get your user access token and organization identifier
+To allow the GX Agent to connect to your GX Cloud organization, you need to supply a user access token and organization identifier.
+
+> **Create a user access token**
+> 1. In GX Cloud, click **Settings** > **Tokens**.
+> 1. In the **User access tokens** pane, click **Create user access token**.
+> 1. In the **Token name** field, enter a name for the token and click **Create**.
+> 1. Copy the user access token value and store it in a safe location - you'll need it shortly!
+>
+>   *You are shown the newly created token once. If you close the dialog before recording the user access token, you'll need to create it again.*
+
+<img src="../common/img/user_token.png" alt="User access token creation results" style="width:400px;"/><br>
+
+> **Get your organization identifier**
+>
+> Copy the value in the **Organization ID** field and store it with the user access token.
+
+<img src="../common/img/organization_id.png" alt="Organization id" style="width:800px;"/><br>
+
+### Start the GX Agent
+You use Docker Compose to start and run the GX Agent and Postgres database.
+
+> **Run the GX Agent and Postgres database using Docker Compose**
+> 1. Clone this workshop repo to your local machine.
+>  ```bash
+> git clone git@github.com:greatexpectationslabs/workshops.git
+> ```
+> 2. `cd` to the `get_started_with_gx_cloud_and_postgres` subdirectory of the cloned repo.
+> 3. Using the following command, replace `<your-organization-id>` and `<your-access-token>` with the values of your GX Cloud organization id and access token, respectively. Execute the command in your terminal.
+>  ```bash
+> GX_CLOUD_ORGANIZATION_ID="<your-organization-id>" GX_CLOUD_ACCESS_TOKEN="<your-access-token>" docker compose up
+> ```
```

Comment:
Should we add a little blurb here to indicate how users will know they've executed this correctly?

Reply:
Good call - I've added this, as well as instructions at the end on how to stop the running Docker Compose.